API-Documentation.md

```diff
@@ -369,20 +428,46 @@ Possible values for `payoutType`:
 | `vendor`             | `string` | The vendorID of the vendor that will charge the payment                         | no       |
-| `items`              | `array`  | Array of [Items](#item) objects that are being purchased         | no       |
+| `items`              | `array`  | Array of [Items](#item) objects that are being purchased         				| no       |
 | `fundTxData`         | `object` | Object containing information about the funding of an shopper                   | no       |
 | `genericTransactions`| `array`  | Objects containing information about every transaction that should be executed  | no       |
-| `totalAmount`        | `decimal` | The total amount of the payment, represented in the selected currency          | no       |
+| `paymentDetails`     | `object` | [Payment Details](#payemnt-details) object  									| no       |
 
 
 ### Item
 
 | Attribute            | Type     | Description                                                                     | Nullable |
 | -------------------- | -------- | ------------------------------------------------------------------------------- | -------- |
 | `description`        | `string` | Description of the item that is being bought. This information is displayed in the invoice. | no       |
-| `lineAmount`             | `decimal`| Fiat amount of the item excluding VAT amount.                                                         | no       |
+| `lineAmount`         | `decimal`| Fiat amount of the item excluding VAT amount.                                   | no       |
 | `quantity`           | `integer`| Quantity of the items                                                           | no       |
 
+
+### Payment Details
+
+| Attribute       | Type      | Description                                                                     | Nullable |
+| --------------- | --------  | ------------------------------------------------------------------------------- | -------- |
+| `taxRate`       | `decimal` | Value added tax rate for payments in Europe countries 							| yes      |
+| `taxAmount`     | `decimal` | Value added tax fiat amount for payments in Europe countries                    | yes      |
+| `baseAmount`    | `decimal` | Payment amount without tax                                                      | no       |
+| `totalAmount`   | `decimal` | Payment amount with tax (if any)      											| no       |
+| `cardHolder`    | `object`  | [Card Holder](#card-holder) object                                              | no       |
+
+
+
+### Card Holder
+
+| Attribute       | Type      | Description                                                                     | Nullable |
+| --------------- | --------  | ------------------------------------------------------------------------------- | -------- |
+| `vatNumber`     | `string`  | For companies in Europe 														| yes      |
```

Comment:
It is not for companies in Europe only. Thats the reason we had a discussion, it could be taxId for countries outside Europe.

you can change it to:
``vatNumber` of the card holder